sass/type/_fonts.scss

```diff
@@ -22,6 +22,8 @@
 // ---------------
 /// Retrieve and normalize font data from the `$fonts` map
 ///
+/// @since 4.0.0 -
+/// - BREAKING: Renamed `font`
```

Comment:
Might be good to include the previous name here, so it's easy to search/replace

Reply:
good catch. this was a mistake. This function was not renamed. Removed the note.